src/main/kotlin/org/simplemc/simpleannounce/config/DurationSerializer.kt

```diff
@@ -0,0 +1,12 @@
+package org.simplemc.simpleannounce.config
+
+import com.fasterxml.jackson.core.JsonGenerator
+import com.fasterxml.jackson.databind.SerializerProvider
+import com.fasterxml.jackson.databind.ser.std.StdSerializer
+import kotlin.time.Duration
+
+class DurationSerializer : StdSerializer<Duration>(Duration::class.java) {
```

Comment:
This class isn't strictly necessary to read (deserialize) the config, but including it is helpful for testing. For example, to create the default config I constructed a `SimpleAnnounceConfig` object and got the yaml out via a test class with `println(objectMapper.writeValueAsString(config))`